src/org/daisy/dotify/formatter/impl/core/TocEntryOnResumedRange.java

```diff
@@ -0,0 +1,28 @@
+package org.daisy.dotify.formatter.impl.core;
+
+/**
+ * Class for the range of toc-entry-on-resumed elements
+ * This class stores a ranges of the form [startRefId,endRefId) or [startRefId,)
+ * 
+ * @author Paul Rambags
+ */
+public class TocEntryOnResumedRange {
```

Comment:
This class would fit nicely in the API I think.

Reply:
Moved to the API. Added some unit tests as well in the API.